src/views/teachers/faq/faq.jsx

```diff
@@ -20,6 +20,8 @@ var TeacherFaq = injectIntl(React.createClass({
                         <dl>
                             <dt><FormattedMessage id='teacherfaq.teacherWhatTitle' /></dt>
                             <dd><FormattedHTMLMessage id='teacherfaq.teacherWhatBody' /></dd>
+                            <iframe width="565" height="318" src="https://www.youtube.com/embed/7Hl9GxA1zwQ"
+                            frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen>...</iframe>
```

Comment:
Thanks so much for helping out with this @St19Galla! Would you please update the `<iframe>` here to not include the `...`?